Userland/Libraries/LibJS/Bytecode/Interpreter.cpp

```diff
@@ -453,4 +451,18 @@ Realm& Interpreter::realm()
     return *m_vm.current_realm();
 }
 
+void Interpreter::push_register_window(Variant<NonnullOwnPtr<RegisterWindow>, RegisterWindow*> window, size_t register_count)
+{
+    m_register_windows.append(move(window));
+    this->window().registers.resize(register_count);
```

Comment:
Should these use fallible `try_append` and `try_resize`?

Reply:
tbh I'm starting to give up on the idea that checking for OOM in minor allocations like this is actually valuable, since making any kind of meaningful recovery is basically impossible.

Unfortunately, we've dug in pretty deep and made a mess in LibJS, LibWeb, LibGUI, and all kinds of apps. :/

Reply:
fair enough - is there an opt out for fallible allocations planned in jakt or will it use something along the lines of `MUST(append())`?

Reply:
I don't know! I haven't really verbalized the thought before just now, but I think maybe this is something better solved with separate containers for OOM-pedantic vs OOM-crashy environments.

I'd like to get back to highly readable no-nonsense code. Feels like we've drifted away from that in userland over the last two years. Totally my fault, I thought it was going to be nice everywhere, but losing faith 😅